data/json/items/gun/40mm.json

```diff
@@ -85,13 +86,11 @@
     "bashing": 12,
     "to_hit": -3,
     "material": [ "steel", "wood" ],
-    "symbol": "(",
     "color": "dark_gray",
-    "skill": "launcher",
     "ammo": "40mm",
     "dispersion": 300,
     "sight_dispersion": 150,
-    "aim_speed": 0,
```

Comment:
This should stay as `aim_speed 0`
Currently `aim_speed` is broken and this doesn't fix it